s4e-backend/src/test/java/pl/cyfronet/s4e/controller/OverlayControllerTest.java

```diff
@@ -259,15 +268,26 @@ public void deleteInstitutionalOverlayShouldBeSecured() throws Exception {
 
     @Test
     public void shouldDeleteInstitutionalOverlay() throws Exception {
+        val oldInstitutionalOverlaysSize = wmsOverlayRepository
+                .findAllInstitutional(
+                        institutionAdmin.getId(),
+                        AppRole.GROUP_MEMBER.name(),
+                        OverlayOwner.INSTITUTIONAL.name()
+                )
+                .size();
         val URL = API_PREFIX_V1 + "/institutions/{institution}/overlays/{id}";
         mockMvc.perform(delete(URL, institution.getSlug(), wmsInstitutionalOverlay.getId())
                 .contentType(MediaType.APPLICATION_JSON)
                 .with(jwtBearerToken(institutionAdmin, objectMapper))
         ).andExpect(status().isOk());
 
         val institutionalOverlays = wmsOverlayRepository
-                .findAllByOwnerTypeAndInstitutionId(OverlayOwner.INSTITUTIONAL, institution.getId(), WMSOverlay.class);
-        assertEquals(institutionalOverlays.size(), 0);
+                .findAllInstitutional(
+                        institutionAdmin.getId(),
+                        AppRole.GROUP_MEMBER.name(),
+                        OverlayOwner.INSTITUTIONAL.name()
+                );
+        assertEquals(institutionalOverlays.size(), oldInstitutionalOverlaysSize - 1);
```

Comment:
void assertEquals(int expected, int actual)

Reply:
Why all kinds? I want to check the type I just created.

Reply:
institutionalOverlays-> actual
oldInstitutionalOverlaysSize - 1 ->expected
you got them reversed